app/models/assignment.rb

```diff
@@ -1,4 +1,9 @@
 class Assignment < ApplicationRecord
   belongs_to :client
   belongs_to :volunteer
+  belongs_to :user
```

Comment:
for my understanding, why did we add here the user? for accessing the user_id right? is it better to access it directly via the user instead the user_id from the client/volunteer?